docker/demo/config/test-suite/complex-dag-mor.yaml

```diff
@@ -13,107 +13,72 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-dag_name: complex-dag-mor.yaml
+dag_name: cow-long-running-example.yaml
```

Comment:
@nsivabalan This dag is is being named as cow-long-running-example but it is only running 1 round, is this intended ?